### ISSUE TITLE
Question: Add class name to dropdown select

### ISSUE THREAD

**Issue by sszdh** [stcr/subscribe-to-comments-reloaded#176]
Hi there,

How can I add a custom class name to the dropdown `<select>` ?

For example I want to have something like this:

```html
<select name="subscribe-reloaded" id="subscribe-reloaded" class="form-control">
.... 
</select>
```

Thanks.

**Comment by sszdh** [comment]
Sad,
I think according to [this](https://github.com/wp-plugins/send-email-only-on-reply-to-my-comment/blob/master/send-email-only-on-reply-to-my-comment.php#L106), there is no way to define any additional `class` name to `<select>` box!
Am I correct? 

Thanks.

**Comment by raamdev** [comment]
@sszdh You don't need to add a class; you can just reference the existing ID:

```css
select#subscribe-reloaded {
/* Styles here */
}
```

**Comment by sszdh** [comment]
@raamdev, Yeah I know, but consider you want using `bootstrap` for UI,  then we should dublicate and redefine each Styles!? Don't we?

In my provided example, the `form-control` class is a defined style for form inputs in `bootstrap` CSS package.

Thanks

**Comment by sszdh** [comment]
I think it is good to have a way to add user-defined  class name to the controls

**Comment by Reedyseth** [comment]
> I think it is good to have a way to add user-defined class name to the controls

@sszdh Yes this will allow the user to have more control over the style of the dropdown. As for now you could add that class manually and then in the future add the option to add a class to the dropdown.

**Comment by sszdh** [comment]
@Reedyseth Thanks for your reply.

**Comment by Reedyseth** [comment]
> Thanks for your reply.

@sszdh You're Welcome !
